UnityProject/Assets/Scripts/Messages/Client/RequestBuildMessage.cs

```diff
@@ -76,47 +85,67 @@ public override void Process(NetMessage msg)
 				}
 			}
 
-			//build and consume
-			void ProgressComplete()
+			if (entry.OnePerTile)
```

Comment:
Not nessarcary, but could remove these brackets and put this on one line like:
if(entry.OnePerTile) msg.Number = 1;

Just to clean it up visually.

Reply:
Cool will do

Reply:
we agreed on only doing that when returning